src/treatment.js

```diff
@@ -41,6 +41,20 @@ class Treatment extends React.Component {
   }
 }
 
+const isRender = (props, propName, componentName) => {
```

Comment:
Well done!

What do you think about renaming the function so it reflects what it's doing? I'm thinking something like `functionOrEmpy`.

Reply:
Yep, sounds good